build-in-container/README.md

```diff
@@ -6,11 +6,17 @@ Please install docker on your system before using the tool.
 ## Usage
 The mariner-docker-builder.sh script presents these options <br />
 <pre>
-  -t               creates container image <br />
-  -b [mariner_dir] creates container, builds specs under [mariner_dir]/SPECS/, & places output under [mariner_dir]/out/ <br />
-  -i [mariner_dir] creates an interactive Mariner build container <br />
-  -c [mariner_dir] cleans up Mariner workspace at [mariner_dir], container images and instances <br />
-  --help           shows help on usage <br />
+  -t                        creates container image <br />
+  -b                        creates container, builds specs under [mariner_dir]/SPECS/, & places output under [mariner_dir]/out/ <br />
```

Comment:
for `-b` `-i` `-c`, are these user facing interface changes since we're dropping `mariner_dir`, or internal interface changes?

Reply:
This is user facing, because now we have multiple optional arguments, instead of just mariner_dir

Reply:
Does this new scheme maintain compat with the previous user interface options? i.e., is `mariner_dir` always still parsed as the first argument?

For user-facing changes to tools, it is important to not break the user facing contract. Users could have built scripts/tools around this tool, and once they take this update, if the user interface changes, their usage/tools would begin failing. Hence why I ask about user-facing options in particular.

Reply:
No, mariner_dir will not be parsed as an argument unless passed with the --mariner_dir flag now. 
Yes, it will break the user's tools if they have built something around this already, but my understanding is that this is the first version, and they have not used it at all since it's lacking the basic feature they need.

Reply:
> but my understanding is that this is the first version, and they have not used it at all since it's lacking the basic feature they need.

This is open-source tooling so we may not know the full set of users who could already be using it.

At minimum, this user interface change should be highlighted in the PR summary, but can you see if there is a way to avoid breaking the compat altogether?

Reply:
Yes, let me add the description to the PR, thanks for highlighting this